src/main/java/com/google/devtools/build/lib/bazel/BazelStartupOptionsModule.java

```diff
@@ -33,7 +33,11 @@
         valueHelp = "<path>",
         help =
             "The location of the user .bazelrc file containing default values of "
-                + "Bazel options. If unspecified, Bazel uses the first .bazelrc file it finds in "
+                + "Bazel options. This option can also be chained together.\n"
```

Comment:
I think you mean "can be specified multiple times"? "Chained together" might be the source of confusion in https://github.com/bazelbuild/bazel/pull/12740#issuecomment-762724352

Reply:
Thanks. Will clarify.

Reply:
adjusted.